.github/pull_request_template.md

```diff
@@ -30,7 +30,51 @@ _Include relevant screenshots_
 
 #### After
 
-### Checklist:
+### Review your work
+Please self review your work before requesting further review. _Wait until you reach the next screen to check off the boxes._
+
+#### Preview
+
+- [ ] Pull down the branch and run the code locally or use Preview build
```

Comment:
I don't think the dev opening the PR would need to pull down their branch since it was developed locally? But def love the second half of this point

Reply:
Yeah makes sense, I def just did a quick cut-and-pasties